site/users/index.html.js

```diff
@@ -48,6 +48,16 @@ var logos = [
     img: 'reindex.png',
     link: 'https://www.reindex.io/'
   },
+  {
+    name: 'Meteor',
+    img: 'meteor.png',
+    link: 'https://medium.com/apollo-stack/snappier-uis-with-apollo-client-graphql-bffab0b52b79#.sls7hdvud'
```

Comment:
Cool. I think we should leave this meteor entry as is, and just omit the Apollo one since Apollo deserves to be highlighted in the /code/ section rather than here.